lib/active_model/serializer.rb

```diff
@@ -27,12 +27,12 @@ class Serializer
       )
     /x
 
-    class << self
-      attr_accessor :_attributes
-      attr_accessor :_attributes_keys
-    end
-
     with_options instance_writer: false, instance_reader: false do |serializer|
+      class_attribute :_type, instance_reader: true
+      class_attribute :_attributes
+      self._attributes ||= []
+      class_attribute :_attributes_keys
```

Comment:
style-wise, should the assignments be right next to eachother, rather than separated by a class_attribute definition?